src/k2/cmd/ycsb/dataload.h

```diff
@@ -0,0 +1,104 @@
+/*
+MIT License
+
+Copyright(c) 2020 Futurewei Cloud
+
+    Permission is hereby granted,
+    free of charge, to any person obtaining a copy of this software and associated documentation files(the "Software"), to deal in the Software without restriction, including without limitation the rights to use, copy, modify, merge, publish, distribute, sublicense, and / or sell copies of the Software, and to permit persons to whom the Software is furnished to do so, subject to the following conditions :
+
+    The above copyright notice and this permission notice shall be included in all copies
+    or
+    substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS",
+    WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT.IN NO EVENT SHALL THE
+    AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM,
+    DAMAGES OR OTHER
+    LIABILITY,
+    WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+    OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+    SOFTWARE.
+*/
+
+#pragma once
+
+#include <vector>
+
+#include <k2/appbase/Appbase.h>
+#include <k2/appbase/AppEssentials.h>
+#include <k2/module/k23si/client/k23si_client.h>
+#include <k2/transport/RetryStrategy.h>
+#include <seastar/core/sleep.hh>
+
+#include "data.h"
+#include "Log.h"
+
+using namespace seastar;
```

Comment:
Please scope the code in the k2 namespace, and avoid "using" in header files in general. I suggest prefixing seastar:: to all seastar usage here as this is what we do in the rest of the codebase